widgetssdk/src/main/java/com/glia/widgets/chat/ChatView.java

```diff
@@ -289,7 +281,8 @@ public void startChat(
                                     contextUrl,
                                     this.theme
                             ),
-                            activity instanceof ChatActivity, activity instanceof ChatActivity && useOverlays
```

Comment:
Revert this, I'll remove it in ChatHead solution

Reply:
it's the second time I run into these warnings and I don't see the reason to keep it. If you need that code for reference, you can always use git history.